Validation/HcalDigis/src/HcalDigisValidation.cc

```diff
@@ -797,10 +797,14 @@ template<class Digi> void HcalDigisValidation::reco(const edm::Event& iEvent, co
 	    
 	    //Special for HF
 	    if (isubdet == 4 && v_ampl[1] > 30. && depth == 1) {
-	      double fBin5 = tool[2] - calibrations.pedestal((*digiItr)[2].capid());
-	      double fBin67 = tool[3] + tool[4]
-		- calibrations.pedestal((*digiItr)[3].capid())
-		- calibrations.pedestal((*digiItr)[4].capid());
+              int soi = (tool.presamples() == 1 ? 1 : 2);
```

Comment:
No reason, I had coded it this way because it was a choice between two different behaviors. I then saw this alternative, but decided to leave it as is. If you hold off triggering tests, I'll change the code.